vendor/bat-native-ads/src/bat/ads/internal/ads_impl.h

```diff
@@ -127,7 +127,8 @@ class AdsImpl : public Ads {
   void SetConfirmationsIsReady(
       const bool is_ready) override;
 
-  std::map<uint64_t, std::vector<AdsHistory>> GetAdsHistory() override;
+  std::map<uint64_t, std::vector<AdsHistory>> GetAdsHistory(
+      const Ads::AdHistoryFilterType adsHistoryFilterTypre) override;
```

Comment:
Nit: typo in function parameter name :-)

Reply:
`adsHistoryFilterTypre` should use `camel_case`

Reply:
Done